dotCMS/src/main/webapp/html/ng-contentlet-selector.jsp

```diff
@@ -128,7 +128,8 @@
             name: "create-contentlet-from-edit-page",
             data: {
                 url,
-				contentType
+				contentType,
+                isNew: true
```

Comment:
Why we need this?

Reply:
I added it testing something. Will remove